test/androidx/app/src/main/assets/www/whitelist/index.html

```diff
@@ -27,7 +27,7 @@
       <script type="text/javascript" charset="utf-8" src="../main.js"></script>
```

Comment:
Maybe this file should be in the folder `www/allowlist/index.html` instead of  `www/whitelist/index.html`?

Reply:
Yes, the renaming of this directory was forgotten. I applied fix.